src/modules/dashboard/sagas/colonyCreate.ts

```diff
@@ -413,59 +406,11 @@ function* colonyCreate({
       );
     }
 
-    /*
-     * Deploy OldRoles
-     */
-    yield put(transactionReady(deployOldRoles.id));
-
-    const {
-      payload: {
-        transaction: {
-          receipt: {
-            logs: [deployOldRolesLog],
-          },
-        },
-      },
-    } = yield takeFrom(
-      deployOldRoles.channel,
-      ActionTypes.TRANSACTION_SUCCEEDED,
-    );
-    const oldRolesAddress = parseExtensionDeployedLog(deployOldRolesLog);
-
-    /*
-     * Set OldRoles role
-     */
-    yield put(
-      transactionAddParams(setOldRolesRole.id, {
-        address: oldRolesAddress,
-      }),
-    );
-    yield put(transactionReady(setOldRolesRole.id));
-    yield takeFrom(setOldRolesRole.channel, ActionTypes.TRANSACTION_SUCCEEDED);
-
     /*
      * Deploy OneTx
      */
     yield put(transactionReady(deployOneTx.id));
 
-    const {
-      payload: {
-        transaction: {
-          receipt: {
-            logs: [deployOneTxLog],
-          },
-        },
-      },
-    } = yield takeFrom(deployOneTx.channel, ActionTypes.TRANSACTION_SUCCEEDED);
-    const oneTxAddress = parseExtensionDeployedLog(deployOneTxLog);
```

Comment:
We still need all of this section here to wait for the transaction, and then get the `oneTxAddress` so that we can set the params of the roles transactions below.